examples/nextjs-auth-demo/pages/api/callback.js

```diff
@@ -0,0 +1,10 @@
+import oidc from '../../utils/oidc';
+
+export default async function callback(req, res) {
+  try {
+    await oidc.handleCallback(req, res, { redirectTo: '/' });
+  } catch(error) {
+    console.error(error)
```

Comment:
Can we show an example of a custom error page here?

Reply:
@sandrinodimattia what about a redirect in case of an error?